README.md

```diff
@@ -2,6 +2,8 @@
 
 This repository contains the Puppet modules for profiles related to WSO2 API Manager and API Manager Analytics.
 
+These modules are supported for Puppet 5.4.0
+
```

Comment:
Shall we add something similar to what we have in our Ansible repos?
![image](https://user-images.githubusercontent.com/15902071/62027458-5d325c80-b1fb-11e9-9676-f421f5b7c9b9.png)

Reply:
Will do